scarlet2/frame.py

```diff
@@ -35,7 +36,8 @@ def C(self):
     @property
     def pixel_size(self):
         if self.wcs is not None:
-            return get_pixel_size(get_affine(self.wcs)) * 60 * 60  # in arcsec
+            # return get_pixel_size(get_affine(self.wcs)) * 60 * 60  # in arcsec
+            return get_scale(self.wcs.wcs).mean() * 60 * 60 # in arcsec
```

Comment:
moving the scale computation to `measure` is not my preferred choice. `measure` should be things we measure from scarlet models, these functions refer to sizes and angles from WCS. I suggest moving them to `frame.py` instead.

Also, the call `get_affine` is to catch different WCS formatting conventions. See https://github.com/pmelchior/scarlet/pull/286 for that issue in scarlet1.